registration/settings.py

```diff
@@ -182,3 +183,17 @@
     STATIC_ROOT = STATIC_DIR
 else:
     STATICFILES_DIRS = [STATIC_DIR]
+    # Uncomment the following to log the SQL queries
+    # LOGGING = {
+    #     'version': 1,
+    #     'disable_existing_loggers': False,
+    #     'handlers': {
+    #         'console': {
+    #             'level': 'DEBUG',
+    #             'class': 'logging.StreamHandler',
+    #         },
+    #     },
+    #     'loggers': {
+    #         "django.db.backends": {"level": "DEBUG", "handlers": ["console"]},
+    #     },
+    # }
```

Comment:
I would let it: it can be really interesting to see the SQL queries happening in live.

For instance, when using a big fixture, it may allow to spot some repeated queries (were some better eager loading should be used).

Reply:
Ok